### ISSUE TITLE
Access toolbox features programmatically without the need of the toolbox

### ISSUE THREAD

**Issue by DeadManPoe** [apache/incubator-echarts#10274]
### What problem does this feature solve?
The problem of accessing toolbox functionalities(i.e, area data zoom) without the need to use the toolbox itself. Now is possible to dispatch actions that map to toolbox functionalities(takeGlobalCursor) but these actions require the actual presence of the toolbox itself. The goal is to set by the default the behaviour of each graph to be similar to what Grafana offers: the exploration of data point but also the natural possibility of immediately select a portion of the graph and automatically zoom into it.

### What does the proposed API look like?
The API can remain the same as the one already offered by echarts(dispatching takeGlobalCursor actions) but without the need to have a toolbox introduced as part of the 'option' object.

**Comment by deqingli** [comment]
If you only want to data zoom, you can use the [dataZoom](https://www.echartsjs.com/option.html#dataZoom) component instead of the toolbox.

**Comment by DeadManPoe** [comment]
I want to do area data zoom which is different from a simple data zoom that you can do by using the data zoom action. I want to be able to select an area on the graph and then zoom inside after the selection is completed; to accomplish this now you can only resort to the toolbox data zoom feature or dispatch the takeGlobalCursor of type 'dataZoomSelect' action, but you cannot dispatch the action without having the toolbox, otherwise, the action will not produce anything.

**Comment by Cloud11PL** [comment]
Any updates regarding this issue?

**Comment by develar** [comment]
That's really a pity. Mouse zoom is working, but no simple way to do enable area data zoom.

**Comment by simonedavico** [comment]
I agree, there should be an option to disable the default feature icons but leave the API enabled to be controlled by `dispatchAction` calls.

**Comment by winsmith** [comment]
I would love to see this feature because I want to enable image download, but provide my own UI for it.

**Comment by basickarl** [comment]
Bump! This is needed!

**Comment by basickarl** [comment]
This should get fixed.

Workaround - I've currently been able to remove the icons with the following set for the icons in the toolbox:

```
  icon: {
    zoom: 'path://',
    back: 'path://',
  },
```

**Comment by mrwunderbar666** [comment]
> This should get fixed.
> 
> Workaround - I've currently been able to remove the icons with the following set for the icons in the toolbox:
> 
> ```
>   icon: {
>     zoom: '-',
>     back: '-',
>   },
> ```
> 
> Any string will work.

Thanks for the hack!
